proposed/2023/Add-dotnet-search.md

```diff
@@ -41,23 +44,91 @@ This option will specify a list of sources to search from. If a source is not sp
+        | Newtonsoft.Json.Schema                      | 3.0.15         |         | 39,648,430      |
+        | Microsoft.AspNetCore.Mvc.NewtonsoftJson     | 7.0.12         |         | 317,067,823     |
+        ...
 
 - Using ``dotnet package search Newtonsoft.Json --exact-match`` on the other side will have the following output
 
-Source: nuget.org
-| Package ID                                  | Latest Version | Authors | Downloads       |
-|---------------------------------------------|----------------|---------|-----------------|
-| Newtonsoft.Json                             | 13.0.3         |         | 3,829,822,911   |
+        Source: nuget.org
+        | Package ID                                  | Latest Version | Authors | Downloads       |
+        |---------------------------------------------|----------------|---------|-----------------|
+        | Newtonsoft.Json                             | 13.0.3         |         | 3,829,822,911   |
+
+#### **Option `--format`**
+
+This option will allow the specification of the output format. The option will take one of these three arguments: Table, List, or Json. The outputs will depend on verbosity. The following examples are for normal verbosity.
+
+- `--format Table` the output will be in a tabular form as shown below
+
+        Source: nuget.org
+        | Package ID                                  | Latest Version | Authors | Downloads       |
+        |---------------------------------------------|----------------|---------|-----------------|
+        | Newtonsoft.Json                             | 13.0.3         |         | 3,829,822,911   |
+- `--format List` : each package will be listed line by line
+
+        >Newtonsoft.Json | 13.0.3 | Authors: N/A | Downloads: 3,829,822,911
+        >Microsoft.IdentityModel.JsonWebTokens | 7.0.3 Authors: N/A | Downloads: N/A
+
+- `--format Json` : A json output will be outputted. It will contain the following metadata.
+  - Source
+  - Package ID
+  - LatestVersion
+  - Authors
+  - Downloads
+
+  The output will take the following format
+
+        [
+            {
+                "source": "source Name1",
+                "packagesList": [
+                    {
+                        "author": "",
+                        "downloads": "",
+                        "latestVersion": "",
+                        "packageId": ""
+                    }
+                ]
+            },
+            {
+                "source": "source Name2",...
+            }
+        ]
+
+#### Option `--verbosity`
+
+  controls the amount of information included in the output of a command. This allows you to tailor the output to your needs, providing either a concise summary or a detailed breakdown of the information returned by the command. 
+
+Here are the details of each verbosity level:
+
+- **`--verbosity quiet`** : The output will contain minimal information. For each package, it will display only the following metadata:
```

Comment:
The description says the output will contain minimal information. Shouldn't the log level be minimal?

I think `quiet` is a bit of an odd option for an operation whose primary result is what gets written for the user. 
restore on the other hand, writes an assets file, that's the primary purpose, not the logs themselves.